app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/firestore/FirestoreDatabaseService.kt

```diff
@@ -126,15 +129,12 @@ class FirestoreDatabaseService(
     override suspend fun removePicture(picture: CategorizedPicture) {
         require(picture is FirestoreCategorizedPicture)
         val ref = db.document(picture.path)
-        if(!ref.get().await().exists()) {
+        try {
+            ref.delete()
+        } catch (e: Exception) {
             throw IllegalArgumentException("The database ${this.db} does not contain the picture ${picture.url}")
```

Comment:
That's not consistent with what you do in removeCategory: that exception would get warned instead of converted into an IAE

Reply:
Indeed I seem to have mixed some things up... Thanks